test/test_array_ops.cpp

```diff
@@ -146,6 +171,89 @@ TEST(ArrayOps, PreventGradient) {
   }
 }  // end of op PreventGradient
 
+// Test op: QuantizeV2
```

Comment:
For now these all look good. Since there will be more quantization ops - we probably need to move them to a separate file? (In a separate PR)

Reply:
For now we were following TF's classification (array ops, nn ops etc). Under that schema Q, DQ, QDQ are under array_ops.

Reply:
Sounds good then.